balboa-client-dispatcher/README.md

```diff
@@ -27,7 +27,7 @@ SBT:
  ```
  libraryDependencies ++= Seq(
    ...
-   "com.socrata" % "balboa-dispatcher-client" %% "[0.15, )",
+   "com.socrata" % "balboa-dispatcher-client" %% "[0.16, )",
```

Comment:
And then let's pick one PR or the other that will change the dependency lines to the form
`"com.socrata" %% "balboa-client-dispatcher" % "0.16.+",`
(updated names, versions, and syntax)